frontends/ol-components/src/components/TiptapEditor/extensions/node/ArticleByLineInfoBar/ArticleByLineInfoBar.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import { NodeViewWrapper } from "@tiptap/react"
+import type { ReactNodeViewProps } from "@tiptap/react"
+import styled from "@emotion/styled"
+import Container from "@mui/material/Container"
+import { RiShareFill } from "@remixicon/react"
+import { useUserMe } from "api/hooks/user"
+import Avatar from "@mui/material/Avatar"
+import { ActionButton } from "@mitodl/smoot-design"
+
+const StyledWrapper = styled.div(({ theme }) => ({
+  width: "100vw",
+  maxWidth: "100vw",
+  position: "relative",
+  left: "50%",
+  right: "50%",
+  marginLeft: "-50vw",
+  marginRight: "-50vw",
+  marginBottom: "56px",
+  background: theme.custom.colors.white,
+  padding: "12px 0",
+  alignItems: "center",
+  justifyContent: "space-between",
+  boxShadow: "0 0 5px 0 rgb(0 0 0 / 15%)",
+  border: `1px solid ${theme.custom.colors.lightGray2}`,
+}))
+
+const InnerContainer = styled(Container, {
+  shouldForwardProp: (prop) => prop !== "noAuthor",
+})<{ noAuthor?: boolean }>(({ noAuthor }) => ({
+  display: "flex",
+  justifyContent: noAuthor ? "flex-end" : "space-between",
+  alignItems: "center",
+  "&&": {
+    maxWidth: "890px",
+  },
+}))
+
+const InfoContainer = styled.div({
+  display: "flex",
+  alignItems: "center",
+  gap: "16px",
+})
+
+const NameText = styled.span(({ theme }) => ({
+  ...theme.typography.body2,
+  color: theme.custom.colors.black,
+}))
+
+const InfoText = styled.span(({ theme }) => ({
+  ...theme.typography.body2,
+  color: theme.custom.colors.silverGrayDark,
+}))
+
+const Spacer = styled.div({
+  marginBottom: "56px",
+})
+
+const ArticleByLineInfoBar = ({ node }: ReactNodeViewProps) => {
+  const { authorName, avatarUrl, readTime, publishedDate, editable } =
+    node.attrs
+  const { isFetching: isLoadingUser, data: user } = useUserMe()
+
+  if (editable) {
+    return (
+      <NodeViewWrapper>
+        <Spacer />
+      </NodeViewWrapper>
+    )
+  }
+
+  const author =
+    !isLoadingUser &&
+    (authorName ||
+      (user?.first_name || user?.last_name
+        ? `${user?.first_name || ""} ${user?.last_name || ""}`.trim()
+        : null))
+
+  return (
+    <NodeViewWrapper>
+      <StyledWrapper>
+        <InnerContainer noAuthor={!author}>
+          {author && (
+            <InfoContainer>
+              <Avatar
+                alt={`${user?.first_name} ${user?.last_name}`}
+                src={avatarUrl}
+              >
+                JK
+              </Avatar>
```

Comment:
IMO the alt text is annoying here; the full name follows anyway.

Is JK a fallback? If I have avatarUrl set, I would see that instead?